app/styles/components/_theme-selector.scss

```diff
@@ -145,6 +145,10 @@
 }
 
 [data-theme="dark-ui"] {
+    input[type="time"]::-webkit-calendar-picker-indicator {
```

Comment:
webkit is chromium, what about firefox? does it work there outside the box?

Reply:
Firefox's native time input doesn't contain an icon like Chrome does so we don't need to think about that.